discord-readme.md

```diff
@@ -0,0 +1,76 @@
+- Using welcoming and inclusive language
+- Being respectful of differing viewpoints and experiences
+- Gracefully accepting constructive criticism
+- Focusing on what is best for the community
+- Showing empathy towards other community members
+Full details on https://github.com/EddieJaoudeCommunity/support/blob/main/CODE_OF_CONDUCT.md
+
+**Enforcement:**
+
+Instances of abusive, harassing or otherwise, unacceptable behaviour may be reported by contacting the @moderators . All complaints will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
+
+EddieBotDev (or EddieBot), would be monitoring language used on the discord server, to ensure inclusive and welcoming language in the discord server. EddieBot suggests alternatives and it would be a good practice to edit your message, reflecting the suggestion
+
+NOTE: EddieBot is still in development, and there might be a few words flagged by it which should not have been flagged. Open an issue on the repo and get a green square!
+
+**Information Regarding Channels:**
+TEXT CHANNELS: These consist of the server's main text channels, that are for discourse regarding topics that aren't very tech-specific, and aren't off-topic
+
+-> #general: main text chat
+
+-> #introductions: give yourself an introduction, and familiarising with the community
+
+-> #standup: everyday of code, give us your standup updates! What you did yesterday,what you plan to do today and any blockers?
+
+-> #opensource: discussing interesting open source news and information, that may or may not be related to EddieJaoudeCommunity
+
+-> #hackathons: the prime area to get notified about upcoming hackathons, and discussing about them
+
+-> #help: ask for help or provide help to the members of the community
+
+-> #resources: a reservoir for the vast amounts of resources, that can and will be referred in the community, regarding diverse topics
+
+-> #remote-working: discussion regarding working remotely
+
+-> #ideas: got an idea / suggestion for the community? We love hearing new ideas :nerd: 
+
+-> #soft-skills: these are also hard, so lets discuss
+
+-> #organisation: our community's GitHub org
+
+-> #careers: advice from the members of the community and from experts in the field about your career
+
+-> #first-timers: get :green_square: to get you started in open source on GitHub - opportunities are provided for first-timers
+
+-> #content-share: share web content
+
+-> #virtual-events: events being hosted in the opensource community
+
+-> #bot-chat: discuss bots, esp. regarding development of EddieBot
+
+TECH: Channels regarding specific areas and languages
+->  #git, #cloud, #python, #linux, #javascript, #typescript, #devops, #tech-other, #php, #java, #go, #svelte, #c-langs.

```

Comment:
if we add a new line i think all items are placed on the same line. What do you think?
```suggestion 

->  #git, #cloud, #python, #linux, #javascript, #typescript, #devops, #tech-other, #php, #java, #go, #svelte, #c-langs.
```